sdk/core/azure-core/src/main/java/com/azure/core/util/logging/ClientLogger.java

```diff
@@ -36,6 +39,13 @@
  */
 public class ClientLogger {
     private final Logger logger;
+    private final String className;
+
+    private static final String LOG_MESSAGE_FORMAT = "%s: %s %s%n%s";
+    private static final String LOG_LEVEL_VERBOSE = "VERBOSE";
+    private static final String LOG_LEVEL_INFO = "INFO";
+    private static final String LOG_LEVEL_WARN = "WARN";
+    private static final String LOG_LEVEL_ERROR = "ERROR";
```

Comment:
Could this be a property on the `LogLevel` enum?

Reply:
Need to have toString in LogLevel.

Reply:
Seems like LogLevel.INFO,name() is working.